### PR TITLE
Update yaml files - pin napari < 0.5

### PR DESCRIPTION
**Pull request by anwai98**

### CODE REVIEW

**Comment by anwai98** [comment]
This PR fixes the napari-related pin mentioned at https://github.com/computational-cell-analytics/micro-sam/issues/693.

**Comment by anwai98** [comment]
Okay, some tests are failing to the incompatible border attribute shifts I made recently. I'll revert  the ones blocking the tests.

File: test/test_prompt_generators.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
-import unittest
+id
```
[the remark this answers is not in the document]
Comment (anwai98) [reply test/test_prompt_generators.py]: My bad, the snippet seems to have gotten copy-pasted by mistake. Should be fixed now.

**Review by constantinpape — APPROVED** [review]
Gonna merge once the tests pass.

**Pull request merged**